src/main.rs

```diff
@@ -53,6 +53,9 @@ pub struct Opt {
     /// The folder to scan
     folder: Option<PathBuf>,
     #[structopt(short, long)]
+    /// The directories to ignore
+    ignore: Vec<PathBuf>,
```

Comment:
I have opted to use an `ignore` option but wondering if we would prefer to use `exclude` with shorthand `x`, I have seen that preference voiced in other issues.